### PR TITLE
EVA-3018: Quick fixes 

### PR DESCRIPTION
**Pull request by nitin-ebi**

### CODE REVIEW

File: src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java [hunk 1]
```diff
@@ -44,7 +44,8 @@ protected void parseReport() throws IOException, NullPointerException {
                 parseAssemblyData(line);
             } else if (!line.startsWith("#")) {
                 String[] columns = line.split("\t", -1);
-                if (columns.length >= 6 && columns[5].equals("=")) {
+                if ((columns.length >= 6 && columns[5].equals("=")) ||
```
Comment (tcezard) [inline src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java:47]: In line 102: `chromosomeEntity.setRefseq(columns[6]);` needs to be checked for `na`

Comment (nitin-ebi) [reply src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java]: updated

Comment (nitin-ebi) [reply src/main/java/uk/ac/ebi/eva/contigalias/dus/NCBIAssemblyReportReader.java]: done

**Review by tcezard — APPROVED** [review]
Looks good to me.

**Review by apriltuesday — APPROVED** [review]

**Pull request merged**